userguide/jails.rst

```diff
@@ -817,44 +772,45 @@ available for a jail.
    | Option       | Description                                                   |
    |              |                                                               |
    +==============+===============================================================+
-   | Edit         | Used to modify the settings described in                      |
-   |              | :numref:`Table %s <jail_overview_tab>`.                       |
-   |              | A jail cannot be edited while it is running. The settings can |
+   | EDIT         | Used to modify the settings described in                      |
+   |              | :ref:`Advanced Jail Creation`.                                |
+   |              | A jail cannot be edited while it is running. The settings     |
    |              | can be viewed, but are read only.                             |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Mount        | Open the :guilabel:`Mount Points` list. Select an existing    |
-   | points       | mount point to :guilabel:`Edit` or click |ui-add| to open     |
-   |              | the :guilabel:`Add Mount Point` screen. A mount point         |
+   | MOUNT        | Select an existing                                            |
+   | POINTS       | mount point to :guilabel:`EDIT` or click                      |
```

Comment:
Is "mount points" always two words, or should it be a compound word (mountpoints)?

Reply:
Quick Google search reveals it being used as two words.

Reply:
Interesting, but misses the point.  We need to be consistent about this within our software and documentation.

Reply:
Majority of our docs are using "mount points".